showkase-processor-testing/src/test/resources/ShowkaseProcessorTest/composable_function_with_multiple_preview_functions_compiles/input/Composables.kt

```diff
@@ -0,0 +1,24 @@
+package com.airbnb.android.showkase_processor_testing
```

Comment:
You will need to generate the output of this function as well. You can do that by making this boolean true and running the test locally - https://github.com/airbnb/Showkase/blob/master/showkase-processor-testing/src/test/java/com/airbnb/android/showkase_processor_testing/BaseProcessorTest.kt#L18

This should generate the relevant output files that you can then commit to this PR

Reply:
I might have misunderstood something here, but my understanding was that this first PR just needed to get it not to throw at compile time. So that is why Im not making a ShowkaseMetadata object when processing the annotation. Then I don't think the test will generate an output file. My question is then, should I make this into multiple PRs or should I try to achieve full on support in this one? 😄 wdyt? 

Furthermore, the test flag was a really nice trick! 😄I totally missed that. Would it be nice to add a comment on that in the contribution guide? wdyt? 😄

Reply:
@oas004 Oh I see. It won't generate an output because your input file does not have any @Preview annotation on a method. Maybe just add one to the same file to ensure that it skips multi preview but correctly picks up previews when they are annotated to a function.

Reply:
Oo yeah good idea! Added one in [7ac8862](https://github.com/airbnb/Showkase/pull/255/commits/7ac8862f35526767a4a857aa47bd7c87ebf9e98d) 😄 Does that look okay? 😄